lisa/_assets/kmodules/sched_tp/parsec.h

```diff
@@ -552,6 +559,8 @@ static inline PARSE_RESULT(u8) parse_not_char(parse_buffer *input, u8 c)
 	{                                                                      \
 		parse_buffer __seq_remainder = *input;                         \
 		parse_buffer __seq_unmodified_input = *input;                  \
+		void *private = input->private;                                \
```

Comment:
The availbility of  `private` needs to be documented in the `@body` description in the kerneldoc comment